GVFS/GVFS.Common/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,22 @@
+using System.Reflection;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("GVFS.Common")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("GVFS.Common")]
+[assembly: AssemblyCopyright("Copyright © Microsoft 2017")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+// Setting ComVisible to false makes the types in this assembly not visible 
+// to COM components.  If you need to access a type in this assembly from 
+// COM, set the ComVisible attribute to true on that type.
+[assembly: ComVisible(false)]
+
+// The following GUID is for the ID of the typelib if this project is exposed to COM
+[assembly: Guid("9ea6ff63-6bb0-4440-9bfb-0ae79a8f9ba9")]
```

Comment:
Can some/most of this information be in the Common file?

Reply:
Nit: Also end with newline please.

Reply:
I think a lot of this can be outright deleted actually, if not moved into the Common generated file. Whenever we fully move to .NET Core, I'd like to again revisit this so I'd rather keep this as is for now since this gets us into a healthy state for Early Adopters. Does anyone feel strongly that we should consolidate this now?

I did fix the new lines on the newly created AssemblyInfo.cs files though, thanks for the good catch!

Reply:
I agree completely, I plan to come back to this before it's time to update the copyright string again though :)